src/main/java/org/openhab/binding/zwave/handler/ZWaveSerialHandler.java

```diff
@@ -141,7 +143,7 @@ public void dispose() {
             serialPort.close();
             serialPort = null;
         }
-        logger.info("Stopped ZWave serial handler");
+        logger.debug("Stopped ZWave serial handler");
```

Comment:
I think INFO is fine for this? It's inline with the recommendations.

Reply:
I don't think other bindings are logging this information at INFO level.
Your equivalent log in the initialize() method is DEBUG level.
But ok, I resotre it to info.